mypy/stubtest.py

```diff
@@ -825,6 +825,13 @@ def verify_funcitem(
         if not callable(runtime):
             return
 
+    if isinstance(stub, nodes.FuncDef):
+        stub_abstract = stub.abstract_status == nodes.IS_ABSTRACT
+        runtime_abstract = getattr(runtime, "__isabstractmethod__", False)
+        # The opposite can exist: some implementations omit `@abstractmethod` decorators
+        if runtime_abstract and not stub_abstract:
+            yield Error(object_path, "runtime method is abstract, but stub is not", stub, runtime)
```

Comment:
The signatures of the runtime and the stub are irrelevant for this check, so the default error message here won't be very good. Maybe something like this would be better?

```suggestion
            yield Error(object_path, "runtime method is abstract, but stub is not", stub, runtime, stub_desc="A concrete method", runtime_desc="An abstract method")
```

Reply:
Why not? 🙂